spec/spec.md

```diff
@@ -213,6 +213,15 @@ A [[ref:Conformant Producer]] and [[ref:Conformant Consumer]] implementation may
 
 Examples in this document use the [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/) and the [Decentralized  Identifiers (DIDs)](https://www.w3.org/TR/did-core/) formats for illustrative purposes only; this specification is intended to support any JSON-serializable [[ref:Claim]] format.
 
+## Use Cases
+
+The following is a non-exhaustive list of some use cases which this specification is intended to support.
+
+1. A [[ref:Holder]]'s agent can find a [[ref:Presentation Definition]] and a [[ref:Submission Requirements Object]] in a well-known location, parse them, and use them to filter the [[ref:Holder]]'s credentials for the [[ref:Holder]], who can (interactively or programmatically) select credentials which their agent pack into a [[ref:Presentation Submission]] submitted back to the [[ref:Verifier]] for assessment.
```

Comment:
```suggestion
1. A [[ref:Holder]]'s agent can find a [[ref:Presentation Definition]] in a well-known location, parse its contents, and use them to filter the [[ref:Holder]]'s credentials on behalf of the [[ref:Holder]], who can (interactively or programmatically) select credentials which their agent pack into a [[ref:Presentation Submission]] submitted back to the [[ref:Verifier]] for assessment.
```

- Changes:
  - Proposed striking submission reqs since it's contained in the PD
  - Related wording cleanup
- Questions (possibly for later): 
  - Should we say "wallet or agent"? This might be a can of worms that we might want to address throughout the spec (low pri)
  - I tried to do a bit more cleanup to clarify that the first agent activities (parsing, etc) can be / generally are done programmatically too. Maybe this is very obvious and doesn't need to be said.